tests/e2e/hcp_external_auth_test.go

```diff
@@ -193,5 +193,57 @@ var _ = Describe("Rosacli Testing", func() {
 					}
 				}
 			})
+
+		It("validation should work when create/list/delete idp and user/admin to external_auth_config cluster via ROSA client - [id:71946]",
+			labels.Medium, labels.NonClassicCluster,
+			func() {
+
+				By("Check if cluster is hosted cp cluster and external_auth_providers is enabled")
+				isHostedCP, err := clusterService.IsHostedCPCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				isExternalAuthEnabled, err := clusterService.IsExternalAuthenticationEnabled(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				if !isHostedCP || !isExternalAuthEnabled {
```

Comment:
Do we really need the check on `isHostedCP` if we have the already the `NonClassicCluster` set ?

Reply:
Just added for double confirmation, I'll remove it if you say?

Reply:
Done